CMakeLists.txt

```diff
@@ -161,6 +163,10 @@ if(ENABLE_VULKAN)
     add_definitions(-DRENDERDOC_SUPPORT_VULKAN)
 endif()
 
+if(ENABLE_VRAPI)
+    add_definitions(-DRENDERDOC_SUPPORT_VRAPI)
```

Comment:
Similarly since the vr hooks are always available, we don't need this support variable to know if it was compiled in or not.